opencr/config_production.json

```diff
@@ -1,72 +1,76 @@
 {
-    "auth": {
-        "secret": "CHANGE-IT",
-        "tokenDuration": 5400
+  "auth": {
+    "secret": "CHANGE-IT",
+    "tokenDuration": 5400
+  },
+  "app": {
+    "port": 3000,
+    "installed": false
+  },
+  "mediator": {
+    "api": {
+      "username": "root@openhim.org",
+      "password": "openhim-password",
+      "apiURL": "https://localhost:8080",
+      "trustSelfSigned": false,
+      "urn": ""
     },
-    "app": {
-        "port": 3000,
-        "installed": false
-    },
-    "mediator": {
-        "api": {
-            "username": "root@openhim.org",
-            "password": "openhim-password",
-            "apiURL": "https://localhost:8080",
-            "trustSelfSigned": false,
-            "urn": ""
-        },
-        "register": false
-    },
-    "fhirServer": {
-        "baseURL": "http://hapi-fhir:8080/fhir",
-        "username": "hapi",
-        "password": "hapi"
-    },
-    "elastic": {
-        "server": "http://es:9200",
-        "username": "",
-        "password": "",
-        "max_compilations_rate": "10000/1m",
-        "index": "patients"
-    },
-    "codes": {
-        "goldenRecord": "5c827da5-4858-4f3d-a50c-62ece001efea"
+    "register": false
+  },
+  "fhirServer": {
+    "baseURL": "http://hapi-fhir:8080/fhir",
+    "username": "hapi",
+    "password": "hapi"
+  },
+  "elastic": {
+    "server": "http://es:9200",
+    "username": "",
+    "password": "",
+    "max_compilations_rate": "10000/1m",
+    "index": "patients"
+  },
+  "codes": {
+    "goldenRecord": "5c827da5-4858-4f3d-a50c-62ece001efea"
+  },
+  "structureDefinition": {
+    "uri": "http://openclientregistry.org/fhir",
+    "reportRelationship": "patientreport"
+  },
+  "matching": {
+    "tool": "elasticsearch",
+    "autoMatchPatientWithHumanAdjudTag": false
+  },
+  "clients": [
+    {
+      "id": "openmrs",
+      "displayName": "OpenMRS"
+    }
+  ],
+  "systems": {
+    "CRBaseURI": "http://openclientregistry.org/fhir",
+    "internalid": {
+      "uri": [
+        "http://openmrs.org/uuid"
+      ],
+      "displayName": "OpenMRS UUID"
```

Comment:
We should rename this to **OpenMRS Patient UUID**

Reply:
OK